### DISCUSSION TITLE
faq

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#4] [Question comments]
All about the site 

https://questions.tripos.org/faq

**Comment by AustenLamacraft** [comment]
<details>
<summary>Spoiler text is now available:</summary>
  
You do it like this:
  ```markdown
  <details>
  <summary>Spoiler warning</summary>
  
  Hidden text goes here.
  
</details>
```

Note that you have to leave a blank line after `</summary>`. 

</details>

Also, you can now copy and past $\LaTeX$ from other comments as well as from the questions.

**Reply by thenewtonium** [reply]
<details> <summary>Does LaTeX rendering work in spoilers?</summary>

$\mathcal{YES}$

</details>

**Comment by AustenLamacraft** [comment]
Would anyone be able to let Part IA and IB students know about the site?

**Reply by artie2000** [reply]
Yup, will do

**Reply by callumhobbis** [reply]
I have sent a message about the website to the Part IA and Part IB Discord servers - hopefully this brings some people in!

**Reply by artie2000** [reply]
I've now messaged the Part IB WhatsApp as well

**Comment by callumhobbis** [comment]
For those wondering about editing and deleting: you need to click the timestamp (e.g. "2 hours ago") of a message to go to the GitHub version of the comment. From there, if you made that comment, you should be able to press the three dots icon on the top right of your comment and then press edit/delete.

**Comment by AustenLamacraft** [comment]
Search in comments is now available! Let me know of any issues.

**Reply by artie2000** [reply]
This is super useful, thank you so much!

**Comment by AustenLamacraft** [comment]
[GitHub just turned on $\LaTeX$ rendering](https://github.blog/2022-05-19-math-support-in-markdown/) so now editing comments and correcting typos in the source should be even easier. It's not perfect: compare this question [on the site](https://questions.tripos.org/part-ii/2019-7/) and [on GitHub](https://github.com/tripos-education/maths-tripos-questions/blob/master/src/part-ii/2019-7.md), but a definite improvement.

**Reply by callumhobbis** [reply]
Wow - so soon after this website's launch too!

**Reply by thenewtonium** [reply]
nice

**Comment by ArthurConmy** [comment]
Feature idea: "help" or "incomplete" tag to comments that shows on homepage. There are so many comments now I don't want to click through them all. But if I could see the ones people want help on, I think this would benefit both me and an incomplete solution.

**Reply by AustenLamacraft** [reply]
There are labels in GH discussions but as yet no way to add / edit via the API (Giscus' developer asked for it [here](https://github.com/github/feedback/discussions/15431)). People could always add a "help wanted" label in GH and it could be displayed here, I guess?

**Reply by thenewtonium** [reply]
I don't think we have perms to set labels at the moment?

At least, I can't see any option to.

**Reply by AustenLamacraft** [reply]
Good point. Perhaps upvoting [this question](https://github.com/github/feedback/discussions/15431) is the best we can do for now.

**Reply by AustenLamacraft** [reply]
Another option would be to display any 😕  or perhaps  👀 reaction on the front page

**Reply by callumhobbis** [reply]
The reaction solution might be good: maybe the list of 😕 reactions could have a heading of "help/incomplete" or something similar so casual users know what the 😕 reaction means, for example? Semi-tangent - is the selection of reaction emojis modifiable?

**Reply by thenewtonium** [reply]
One problem with this is that people might not necessarily un-react once there is a solution.
There needs to be some way of overriding once there is a solution, I guess with a "solved" react? (👍 I guess?)

**Reply by AustenLamacraft** [reply]
... or make one of the reactions (say 👀 ) transient and remove it after 24 hours? 

I've opened this as [an issue](https://github.com/tripos-education/maths-tripos-questions/issues/250#issue-1246144572) so maybe we should continue the discussion there?

**Comment by AustenLamacraft** [comment]
Thanks to @thenewtonium for a major reorganisation of the questions!

**Reply by silkca** [reply]
Extremely helpful - thank you!

**Reply by artie2000** [reply]
Oh my god this is ideal tysm @thenewtonium

**Comment by cirosantilli** [comment]
Can I post answers on comments? It appears yes, e.g. https://questions.tripos.org/part-ii/2021-101/ and unsurprisingly answers account for the majority of comments: https://github.com/tripos-education/maths-tripos-questions/discussions

Do I have to put answers in spoilers? More explicitly, will non spoiled answers be taken down by admins? TODO.

Do I own copyright for my answers? Or are answers derivative work of the questions which are presumably copyrighted by the University of Cambridge? The derivative work argument feels compelling.

Spoiler text is against my philosophy BTW, I'd rather have answers without clicking a bunch of stuff, but of course, I'll respect the rules if I ever post here, though that is unlikely because these will likely never show up on natural Google queries outside of Cambridge Uni given that the titles are the question numerical IDs. But it is valuable as a way for me to gauge what the students are learning.

Fantastic initiative for Cambridge Uni students though. Reminds me of back in the day when I was at Uni in Brazil in 2007, and a dude at the photocopy shop would sell photocopies of solved previous exams!

**Reply by AustenLamacraft** [reply]
I didn't have a particular view, but it seems that people like to post answers, and the possibility of spoilers was a requested feature. I think people like to be able to see that an answer is available, without seeing the full solution.

I've never given any thought to whether answers are derivative work. For example, should it matter whether I make a comment ("This one's hard!") or give an answer? My feeling is that it would be hard to make sharp distinction between the two.
